CMSgov/mdct-mcr
# Simplify hydration

## Description
We were running into cases where we had to arbitrarily wrap data in a `fieldData` label to have it parsed correctly. Now we've removed those wrappings and simply passed the data in one layer deeper

### How to test
Fill out the form (make sure to try Standard, Drawer, and ModalDrawer pages).
Hydration works on everything!

### Changed Dependencies
none

## Code author checklist

## Reviewer checklist (two different people)
